core/iwasm/interpreter/wasm_interp_classic.c

```diff
@@ -941,6 +987,21 @@ wasm_interp_call_func_bytecode(WASMModuleInstance *module,
         else if (cache_items[1].start_addr == frame_ip) {
           end_addr = cache_items[1].end_addr;
         }
+#if WASM_ENABLE_DEBUG_ENGINE != 0
+        else if (!wasm_loader_find_block_addr(exec_env,
+                                              (BlockAddr*)exec_env->block_addr_cache,
+                                              frame_ip, (uint8*)-1,
+                                              LABEL_TYPE_BLOCK,
+                                              &else_addr, &end_addr)) {
+#else
+        else if (!wasm_loader_find_block_addr((BlockAddr*)exec_env->block_addr_cache,
+                                              frame_ip, (uint8*)-1,
+                                              LABEL_TYPE_BLOCK,
+                                              &else_addr, &end_addr)) {
+#endif
```

Comment:
Seems the code should be:
```C
#if WASM_ENABLE_DEBUG_ENGINE != 0
        else if (!wasm_loader_find_block_addr(exec_env,
                                              (BlockAddr*)exec_env->block_addr_cache,
                                              frame_ip, (uint8*)-1,
                                              LABEL_TYPE_BLOCK,
                                              &else_addr, &end_addr)) {
          wasm_set_exception(module, "find block address failed");
          goto got_exception;
        }
#endif
```
Only need to handle it when WASM_ENABLE_DEBUG_ENGINE is defined.